contracts/mixnet/src/storage.rs

```diff
@@ -304,6 +306,13 @@ pub fn gateways_owners_read(storage: &dyn Storage) -> ReadonlyBucket<IdentityKey
 }
 
 // delegation related
+pub fn all_mix_delegations_read<T>(storage: &dyn Storage) -> ReadonlyBucket<T>
```

Comment:
Can't we use a concrete type here?

Reply:
Generic type makes it usable for migrations as well, and eliminates the possible mistakes of re-writing a lot of code just for migration deserializations